registry/consul/passing_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/hashicorp/consul/api"
 )
 
+//Add Warning, Unkown
```

Comment:
That's already gone in the last commit. I couldn't figure out how to squash the commits. I can spend more time on that if you wish.